### ISSUE TITLE
🐛 [BUG]  false godmode detected at car rental

### ISSUE THREAD

**Issue by busco82** [bitpredator/empiretown#859]
### Description

bug from vehicle rental
![20241015152919_1](https://github.com/user-attachments/assets/47685c8b-f4f3-45a5-9781-d9daeb00ad8c)

### Reproduction URL

![20241015152919_1](https://github.com/user-attachments/assets/47685c8b-f4f3-45a5-9781-d9daeb00ad8c)

### Reproduction steps

```bash
1. Go to 'busco82'
2. Click on '....'
3. Scroll down to '....'
4. See error kick from anti godmode
when I go to rent the vehicle near the driving school the system gives me the error that I use godmode

(mi devi 20 martelli e 25 bende di cotone :rofl: )
```

### OS

Windows

**Comment by busco82** [comment]
![218_219](https://github.com/user-attachments/assets/f209df58-5d80-43d2-8722-98e741037060)

**Comment by bitpredator** [comment]
I don't understand what the problem is?

**Comment by bitpredator** [comment]
fixed in pull request #858
